app/src/main/res/layout/stats_general.xml

```diff
@@ -14,6 +14,62 @@
             android:layout_gravity="center_horizontal"
             android:orientation="vertical">
 
+            <LinearLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="left"
+                android:orientation="horizontal"
+                android:padding="5dp">
+
+                <TextView
+                    android:id="@+id/textView20"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:paddingStart="5dp"
+                    android:paddingEnd="0dp"
+                    android:text="@string/stats_high_value"
+                    android:textAppearance="?android:attr/textAppearanceMedium" />
```

Comment:
Is there a reason for the smaller font size?

Reply:
>other statistics

The two new values are not statistics.  

I appreciate your point about it getting attention, which is not what I want.  In fact, I want them to take less attention.
Can you think of any other way to accomplish this?

Reply:
IMHO, it is not necessary to distinguish the setting values from the statistic values. If you really want to, maybe one way is to write "Range setting: 3.9 ‒ 10 mmol/L".